onnxruntime/python/tools/transformers/models/whisper/convert_to_onnx.py

```diff
@@ -169,6 +169,69 @@ def parse_arguments(argv=None):
     )
     parser.set_defaults(chain_model=True)
 
+    parser.add_argument(
+        "--extra_decoding_ids",
+        required=False,
+        action="store_true",
+        help="Need extra starting decoding ids for some feature like cross qk. Default if false.",
+    )
+    parser.set_defaults(extra_decoding_ids=False)
+
+    parser.add_argument(
+        "--collect_cross_qk",
+        required=False,
+        action="store_true",
+        help="Beam search model collect stacked cross QK.",
+    )
+    parser.set_defaults(collect_cross_qk=False)
+
+    parser.add_argument(
+        "--output_cross_qk",
+        required=False,
+        action="store_true",
+        help="Beam search model output collected qk as output. Also hint collect_cross_qk",
+    )
+    parser.set_defaults(output_cross_qk=False)
+
+    parser.add_argument(
+        "--no_speech_token_id",
+        default=50362,
+        type=int,
+        help="specify no_speech_token_id. Default is 1000. if >= 0, will be add into beam search attr",
```

Comment:
The default value is different from the description. Can you also check if `no_speech_token_id` is the same or different for the multilingual (e.g. `openai/whisper-tiny`) and English-only (e.g. `openai/whisper-tiny.en`) models?

Reply:
Nice finding. Changed.